libs/feature/search/src/lib/utils/service/fields.spec.ts

```diff
@@ -666,4 +686,41 @@ describe('search fields implementations', () => {
       })
     })
   })
+  describe('UserSearchField', () => {
+    beforeEach(() => {
+      searchField = new UserSearchField(injector)
+    })
+    describe('#getAvailableValues', () => {
+      let values
+      beforeEach(async () => {
+        values = await lastValueFrom(searchField.getAvailableValues())
+      })
+      it('orders results by ascending key', () => {
+        expect(repository.aggregate).toHaveBeenCalledWith({
+          'userinfo.keyword': {
+            type: 'terms',
+            limit: 1000,
+            field: 'userinfo.keyword',
+            sort: ['asc', 'key'],
+          },
+        })
+      })
+      it('returns the available editors, order by ascending key (alphabetical)', () => {
```

Comment:
Looking at your new code, your new test is mostly testing the formatting of the user info. I'm a bit lost as to why we're talking of editors and sorting.

Reply:
True, I've renamed the titles to make it clearer, the sorting is rather a side effect, indeed.